tests/unit/store/elemental-chat.spec.js

```diff
@@ -22,7 +21,8 @@ const makeMessage = (uuid, content) => ({
 })
 
 describe('elementalChat store', () => {
-  it('yam', async () => {
+  // TODO: update test description
+  it.skip('yam', async () => {
```

Comment:
😆  I thought I'd updated this in develop, but apparently not!

```suggestion
  it.skip('handles incoming messages from signals and list_messages', async () => {
```